### ISSUE TITLE
Project Accelerator Licensing Information?

### ISSUE THREAD

**Issue by myatix** [OfficeDev/Project-Accelerator#74]
Hi,

I have been trying to find the licensing requirements for the Project Accelerator solution. There doesn't appear to be any details about this?

I can see the Project Accelerator is built as a Model Driven App on Dataverse so I presume that all users using the App will require a Premium PowerApp license?

Or is it okay to use the App with a Project Operations Team Member license?

We have 2 user types:

Project Managers
Project Members

Licensing all the project members with a Premium PowerApp license could prove to be quite expensive so I was wondering if there was an alternative to this?

Project Members only really need access to see project tasks so a $15 user license seems a little excessive for this solution.

----
Also how do I set security permissions within Project accelerator. When I read the documentation users require the Project Common and Project user permissions, but how do we set permissions for Project member vs project users, vs project requestors?

**Comment by Zajc** [comment]
You can define your own security roles as in any other Model-driven Power App: https://docs.microsoft.com/en-us/project-for-the-web/project-for-the-web-security-roles

**Comment by hmbouk** [comment]
TLDR: You need Microsoft 365 Project Plan licenses and Power BI Pro (or better) licenses to use the solution.

The Project Accelerator solution uses the same Dataverse infrastructure set up by the Project Online service.  You can purchase Project Plan 1, 2 or 3 licenses for your Project Managers.  I have provided a link to the Project Plan options and comparison of features.

https://www.microsoft.com/en-ca/microsoft-365/project/compare-microsoft-project-management-software?&ef_id=Cj0KCQjw08aYBhDlARIsAA_gb0dxyKxbplxdd0PxUTysAGWZbQuamwh69jdqyw172OKgXGZNobl4lpoaAnblEALw_wcB%3aG%3as&OCID=AIDcmmwf9kwzdj_SEM_Cj0KCQjw08aYBhDlARIsAA_gb0dxyKxbplxdd0PxUTysAGWZbQuamwh69jdqyw172OKgXGZNobl4lpoaAnblEALw_wcB%3aG%3as&lnkd=Google_O365SMB_Brand&gclid=Cj0KCQjw08aYBhDlARIsAA_gb0dxyKxbplxdd0PxUTysAGWZbQuamwh69jdqyw172OKgXGZNobl4lpoaAnblEALw_wcB&activetab=tabs%3aprimaryr1

I have two clients where I have installed the Project Accelerator solution and set up all of the Project Managers with Project Plan 3 licenses so they can use the Project Online solution including Roadmaps.  Project Members do not require a Project Plan license in order to view project schedules in Project Online.  Without a Project Plan license, the Project Members can't mark tasks complete, although I have read this will be changing.  I set up only the Project Managers with access to the Project PowerApp from the Project Accelerator solution.  I have found there is way too much detail in the information contained in the Project PowerApp for a typical Project Member.

You will also need to consider whether you want to purchase Power BI Pro licenses so that all of your people, Project Managers and Project Members can have access to the reports provided in the Project Accelerator solution.  In my opinion, I think Project Members are best served by having access to the project schedule and project reports.  The Project Managers will need full access to everything.

Your Project Members are given access to a particular project by adding them as members to a project using the Project Online interface.  The lack of a license controls their access.  If you need/want Project Members to be able to update the project tasks, then you could purchase Project Plan 1 licenses.

I have found the solution to be very flexible, and you can customize it to your own needs.  I have found the reports too overwhelming for anyone other than experienced PMs so you may well want to develop simpler reports.

**Comment by hmbouk** [comment]
Quick update: I just checked the Microsoft 365 Roadmap, and user can update a project task if they have a valid Microsoft 365 license.  This will help keep the number of Microsoft 365 Project Plan licenses to only your PMs.

https://www.microsoft.com/en-us/microsoft-365/roadmap?filters=Microsoft%20Project

See feature ID 88884
